### DISCUSSION TITLE
[craft 5] - provide some way for field instances to inform which field they are instance of

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#13788] [Ideas] [answered]
When we change field handle and label in the field layout, there is actually nothing that informs us which field this specific instance is referring to. We need to open edit slideout and clear handle/label field to see original values.

Would be great if there was some kind of link to original field provided in the slideout, or even better if label of original field was included in field tile, along with overwritten one in the field layout.

**Comment by brandonkelly** [comment, marked as answer]
You can get the global field instance like so:

```php
$globalField = Craft::$app->fields->getFieldById($field->id);
// $globalField->name
// $globalField->handle
// $globalField->uid
```

**Reply by piotrpog** [reply]
@brandonkelly actually what i means is something UX related - so we can see in field layout designer which field instance is instance of. Right now when we change field instance label and handle, its impossible to see it, unless we change these back.
